### PR TITLE
Fix examples after React 16 bump

### PR DESCRIPTION
**Pull request by cdriscol**
Fixes https://github.com/facebook/draft-js/issues/1516

### CODE REVIEW

**Comment by mitermayer** [comment]
Thank you! will test this tomorrow and add to my to merge list!

File: examples/draft-0-10-0/color/color.html [hunk 1]
```diff
@@ -26,8 +26,8 @@
       color may be active at any time.
     </div>
     <div id="target"></div>
-    <script src="../../../node_modules/react/dist/react.js"></script>
-    <script src="../../../node_modules/react-dom/dist/react-dom.js"></script>
+    <script src="../../../node_modules/react/umd/react.production.min.js"></script>
```
Comment (mitermayer) [inline examples/draft-0-10-0/color/color.html]: can we update this PR to use the unminified dependencies ?

File: examples/draft-0-9-1/rich/rich.html [hunk 2]
```diff
@@ -24,8 +24,8 @@
   </head>
   <body>
     <div id="target"></div>
-    <script src="../../../node_modules/react/dist/react.js"></script>
-    <script src="../../../node_modules/react-dom/dist/react-dom.js"></script>
+    <script src="../../../node_modules/react/umd/react.production.min.js"></script>
```
Comment (mitermayer) [inline examples/draft-0-9-1/rich/rich.html]: can we update this PR to not use the production minified dependency ? It's useful to use the dev version for debugging purposes

Comment (mitermayer) [reply examples/draft-0-9-1/rich/rich.html]: the `.development.js`  variants

File: examples/draft-0-10-0/color/color.html (hunk 1 above)
Comment (cdriscol) [reply examples/draft-0-10-0/color/color.html]: yeah, i'll make that change here quick

**Comment by cdriscol** [comment]
@mitermayer - good call.  I made the changes to development versions of react and react-dom

**Pull request closed without merging**